### PR TITLE
Use '/search' endpoint to launch search, allow keywords in search

### PR DESCRIPTION
**Pull request by db3000**
Search works much better with the correct endpoint.

### CODE REVIEW

**Comment by db3000** [comment]
Potential improvements:
- The '/search/browse' endpoint takes an optional 'type' parameter, which include 'tv-show' or 'movie', so possibly Alexa could recognize "search for tv show x" or "search for movie y" and add that parameter to the search. I am worried about it confusing "tv show" for search keywords though. There is also a 'season' parameter, so might be nice to say "search for tv show x season #"
- There also options 'match-any' and 'launch' which let roku launch the search result immediately, so possibly Alexa could recognize "search for x and launch" as a trigger to jump into the first result. Again, it might just pick up "and launch" as part of the keyword, plus this is probably about as good as Google's "I'm Feeling Lucky" button for getting where you want to go.
- You can also specify a 'provider' or 'provider-id', allowing Alexa to recognize "search for x on netflix". Maybe it should be "search netflix for x" so that "on netflix" isn't confused with the keyword.

File: app/__init__.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def homepage():
     return render_template('home.html')
 
 
-@app.route('/alexa/', methods=['POST'])
+@app.route('/alexa', methods=['POST'], strict_slashes=False)
```
Comment (peap) [inline app/__init__.py:37]: Can I ask what this change is for? Just curious; I'm happy to keep it in. Did you change your endpoint to /alexa in the developer portal?

**Comment by db3000** [comment]
It lets you use /alexa or /alexa/ in the address. Without this if you go to /alexa it would send back an HTTP redirect, which Alexa doesn't follow. Not necessary, but I ran into this issue, and it's a bit annoying to diagnose as Amazon doesn't really give you any info when a request fails.

**Comment by db3000** [comment]
Thanks!

**Pull request merged**